src/AeroSharp/AeroSharp.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <Version>1.1.0</Version>
+    <Version>1.2.0</Version>
```

Comment:
I'm happy to discuss whether this is just a minor version change or if we think it should be a major one.

Reply:
New features but no breaking changes so probs minor?

Reply:
I'd agree there!